src/mmrelay/meshtastic/ble.py

```
@@ -0,0 +1,791 @@
+        facade.logger.debug("BleakClient not available for stale connection cleanup")
+    except Exception as e:  # noqa: BLE001 - disconnect cleanup must not block startup
+        # Other errors during best-effort disconnect (e.g., from future.result() or asyncio.run())
+        # are non-fatal; log and continue.
+        facade.logger.debug(
+            "Error during BLE disconnect cleanup for %s",
+            address,
+            exc_info=e,
+        )
+
+
+def _disconnect_ble_interface(iface: Any, reason: str = "disconnect") -> None:
```

Ruff (0.15.9)

[warning] Too many branches (24 > 12)

(PLR0912)